redirects/pages.yml

```diff
@@ -770,7 +770,7 @@
 - from:
     - /docs/multiplatform-mobile-introduce-your-team.html
     - /docs/kmm-introduce-your-team.html
-  to: /docs/multiplatform-introduce-your-team.html
```

Comment:
The most important redirect is missing, from the last page name "/docs/multiplatform-mobile-introduce-your-team.html" to the new location

Reply:
https://github.com/JetBrains/kotlin-web-site/pull/4145 fixed here